specification/columns/contractedcost.md

```diff
@@ -2,6 +2,8 @@
 
 Contracted Cost represents the cost calculated by multiplying [*contracted unit price*](#glossary:contracted-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). Contracted Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on negotiation activities, by comparing it with [List Cost](#listcost). If negotiated discounts are not applicable, the Contracted Cost defaults to the List Cost.
 
+When aggregating for savings calculations, it's important to exclude one-time or recurring charges that are paid to cover future eligible charges or the covered charges themselves. This exclusion helps prevent double counting of these charges in the aggregation. For instance, charges categorized as [ChargeCategory](#chargecategory) "Purchase" and their related [ChargeCategory](#chargecategory) "Tax" charges for a [Commitment-Based Discount](#glossary:commitment-based-discount) might be excluded from an accrual-basis cost aggregation of Contracted Cost. This is because the "Usage" and "Tax" charge records provided during the term of the commitment discount already specify the Contracted Cost.
```

Comment:
```suggestion
When aggregating for savings calculations, it's important to exclude either one-time or recurring charges ([ChargeCategory](#chargecategory)"Purchase") that are paid to cover future eligible charges or the covered charges themselves. Which you exclude depends on if you are aggregating cost on a billed basis (exclude Purchases) or accrual basis (exclude covered charges). This exclusion helps prevent double counting of these charges in the aggregation. For instance, charges categorized as [ChargeCategory](#chargecategory) "Purchase" and their related [ChargeCategory](#chargecategory) "Tax" charges for a [Commitment-Based Discount](#glossary:commitment-based-discount) might be excluded from an accrual-basis cost aggregation of Contracted Cost. This is because the "Usage" and "Tax" charge records provided during the term of the commitment discount already specify the Contracted Cost.
```